samples/openapi3/client/petstore/python-experimental/docs/apis/tags/FakeApi.md

```diff
@@ -2602,7 +2601,6 @@ user list
 ```python
 import petstore_api
 from petstore_api.apis.tags import fake_api
-from petstore_api.model.foo import Foo
```

Comment:
TODO why did this needed import vanish?

Reply:
It vanished because codegenParameter.schema imports were not being added. I fixed it